`bazel run //scenarios:kubernetes_e2e_test` failing locally

Seeing this error:

```
======================================================================
FAIL: test_default_tag (__main__.ScenarioTest)
Ensure the default tag exists on gcr.io.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/justinsb/.cache/bazel/_bazel_justinsb/a1e84e01d786bcdb69b69577a117fc89/execroot/__main__/bazel-out/local-fastbuild/bin/scenarios/kubernetes_e2e_test.runfiles/__main__/scenarios/kubernetes_e2e_test.py", line 397, in test_default_tag
    self.assertNotIn('errors', data)
AssertionError: 'errors' unexpectedly found in {u'errors': [{u'message': u"Manifest with tag 'v20171018-fb8014bc-master' has media type 'application/vnd.docker.distribution.manifest.v2+json', but client accepts '*/*'.", u'code': u'MANIFEST_UNKNOWN'}]}
```

And indeed:
```
> curl https://gcr.io/v2/k8s-testimages/kubekins-e2e/manifests/v20171018-fb8014bc-master
{"errors":[{"code":"MANIFEST_UNKNOWN","message":"Manifest with tag 'v20171018-fb8014bc-master' has media type 'application/vnd.docker.distribution.manifest.v2+json', but client accepts '*/*'."}]}
```

--- a/scenarios/kubernetes_e2e_test.py
+++ b/scenarios/kubernetes_e2e_test.py
@@ -25,7 +25,6 @@ import re
 import shutil
 import string
 import tempfile
-import urllib
 import urllib2
 import unittest
 import time
@@ -393,9 +392,12 @@ class ScenarioTest(unittest.TestCase):  # pylint: disable=too-many-public-method
         self.assertIsNotNone(match)
         url = 'https://gcr.io/v2/%s/manifests/%s' % (match.group(1),
                                                      match.group(2))
-        data = json.loads(urllib.urlopen(url).read())
+        req = urllib2.Request(url)
+        req.add_header('Accept', 'application/vnd.docker.distribution.manifest.v2+json')
+        data = json.loads(urllib2.urlopen(req).read())
         self.assertNotIn('errors', data)
-        self.assertIn('name', data)
+        self.assertIn('config', data)
+        self.assertIn('digest', data.get('config'))
 
     def test_docker_env(self):
         """

